Data/Sys/GameSettings/GLC.ini

```diff
@@ -21,7 +21,6 @@ EmulationStateId = 4
 
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
-DstAlphaPass = True
```

Comment:
@Linktothepast Do you remember why this option is set? I doubt it is required any more...